src/modules/mount/main.py

```diff
@@ -321,6 +327,10 @@ def run():
         return (_("Configuration Error"),
                 _("No partitions are defined for <pre>{!s}</pre> to use.").format("mount"))
 
+    # swap
+    swap_partitions = [p['device'] for p in partitions if p['fs'] == 'linuxswap' ]
```

Comment:
Isn't this the same as before?

We can't use swap partitions the user hasn't chosen to use as part of this install.  It isn't safe to overwrite the data blindly.

Reply:
Same content.
I requested it again as it was not taken into the branch.
Whether the user has configured the partition manually or automatically, the swap partition will not be mounted (swapon) if it does not exist, right?
If the swap does not exist, it will not be processed, so there is no problem.

Reply:
maybe,,,disappeared because of the relation of the merge?

```
    swap_partitions = [p['device'] for p in partitions if p['fs'] == 'linuxswap' and p['claimed'] == False ]
    if ( swap_partitions != [] ):
        enable_swap_partition(swap_partitions)
```

Reply:
It looks like the code isn't quite committed correctly.

You don't need that `if` you added with the most recent commit but you didn't add the other.